From 5d5d5654277c5b8c394763431d2f69b4bea632a9 Mon Sep 17 00:00:00 2001
From: Brad Schoening <bschoeni@gmail.com>
Date: Wed, 4 Sep 2024 22:55:58 -0400
Subject: [PATCH] Updated cython version

---
 setup.py              | 2 +-
 test-requirements.txt | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index 1558512fcf..6948b1af2b 100644
--- a/setup.py
+++ b/setup.py
@@ -368,7 +368,7 @@ def run_setup(extensions):
         # 1.) build_ext eats errors at compile time, letting the install complete while producing useful feedback
         # 2.) there could be a case where the python environment has cython installed but the system doesn't have build tools
         if pre_build_check():
-            cython_dep = 'Cython>=0.20,!=0.25,<0.30'
+            cython_dep = 'Cython>=3.0'
             user_specified_cython_version = os.environ.get('CASS_DRIVER_ALLOWED_CYTHON_VERSION')
             if user_specified_cython_version is not None:
                 cython_dep = 'Cython==%s' % (user_specified_cython_version,)
