diff --git a/cassandra/io/libevwrapper.c b/cassandra/io/libevwrapper.c
index 99e1df30..52d5e8b2 100644
--- a/cassandra/io/libevwrapper.c
+++ b/cassandra/io/libevwrapper.c
@@ -665,9 +665,6 @@ initlibevwrapper(void)
     if (PyModule_AddObject(module, "Timer", (PyObject *)&libevwrapper_TimerType) == -1)
         INITERROR;
 
-    if (!PyEval_ThreadsInitialized()) {
-        PyEval_InitThreads();
-    }
 
 #if PY_MAJOR_VERSION >= 3
     return module;
